ext/srt/gstsrtobject.h

```diff
@@ -69,6 +69,7 @@ struct _GstSRTObject
   GClosure                     *caller_removed_closure;
 
   gchar                        *passphrase;
+  gchar                        *streamid;
```

Comment:
I tended to use `GstStructure` for uri parameters for dumping internal values by GStreamer's debugging macros, but it would be okay if you want to keep stream-id as a special parameter like passphrase.